sql/sql_table.cc

```diff
@@ -9790,6 +9790,8 @@ bool mysql_checksum_table(THD *thd, TABLE_LIST *tables,
 	      row_crc= my_checksum(row_crc, t->record[0], t->s->null_bytes);
             }
 
+	    uchar *checksum_start= NULL;
+	    size_t checksum_length= 0;
```

Comment:
there where a few inconsistencies in that function. corrected for the mysql_checksum_talbe function.